src/components/TopBar/OldUIAnnouncement.tsx

```diff
@@ -0,0 +1,41 @@
+import { Alert, AlertActionCloseButton, AlertVariant } from '@patternfly/react-core';
+import { ApplicationsIcon, ArrowIcon } from '@patternfly/react-icons';
+import { useState } from 'react';
+
+const PNC_OLD_UI_URL_TRIGGER = 'secondary';
+
+export const OldUIAnnouncement = () => {
+  const [isClose, setIsClose] = useState<boolean>(false);
+  const oldUIUrl = process.env.REACT_APP_PNC_OLD_UI_WEB;
+
+  /* Only display the announcement when old ui is set to secondary */
+  const shouldDisplay = !!(oldUIUrl && oldUIUrl.includes(PNC_OLD_UI_URL_TRIGGER));
+
+  if (!shouldDisplay) {
+    return null;
+  }
+
+  /* Temporarily close announcement, will not save to storage in case users may need to switch later*/
+  const closeAnnouncement = () => {
+    setIsClose(true);
+  };
+
+  return isClose ? null : (
+    <Alert
+      id="old-ui-announcement-bar"
+      variant={AlertVariant.custom}
+      customIcon={<ApplicationsIcon />}
+      actionClose={<AlertActionCloseButton onClose={closeAnnouncement} />}
+      title={
+        <>
+          You are currently using PNC new Web UI. If you prefer the old version, you can{' '}
```

Comment:
Welcome to the new PNC Web UI. The [old UI version](http://example.com) is temporarily available, but it will soon be decommissioned.